### DISCUSSION TITLE
Why the GameProcess crached by some exceptions

### DISCUSSION THREAD

**Discussion by ly3027929699** [stride3d/stride#1795] [Q&A] [answered]
i have no idea.Why can't the program continue running?
![_G ~5NW@R8_@WC}GWK{J`WL](https://github.com/stride3d/stride/assets/42492716/fba11f73-2411-4d04-ad1f-0b5a3e822acf)

**Comment by ly3027929699** [comment]
I am very confused because I have never encountered a crash in the game process when writing in Unity before. Such a crash is obviously inconsistent with my experience.

**Comment by ly3027929699** [comment]
I have determined that the error is reported because Prefab cannot be serialized, and because I have used DisplayAttribute, this is the real cause of the crash. I still don't know why this Prefab cannot be loaded.

**Reply by Eideren** [reply]
Can you send your project over ? Or create a small repro

**Reply by ly3027929699** [reply]
https://github.com/ly3027929699/TestStride
i use the scene named MainScene
Thanks!

**Reply by Eideren** [reply, marked as answer]
You just have to add `<PackageReference Include="Stride.Physics" Version="4.1.0.1" />` to your `GameMenu.Game.csproj` below the other `PackageReference`. 
I'll add a check in the engine to avoid those kinds of issues in the future.

**Reply by ly3027929699** [reply]
Now a new error has appeared.Error: Unexpected exception while executing a script.. Stride.Core.Serialization.Contents.ContentManagerException: Unexpected exception while loading asset [MainScene]. Reason: No serializer available for type id 4b3654f50f66f74932ee4d479eb53fc4 and base type Stride.Engine.EntityComponent. Check inner-exception for details.
[error.txt](https://github.com/stride3d/stride/files/12660382/error.txt)

**Reply by ly3027929699** [reply]
I have updated my repos. I have tried reloading the project and restarting the engine, but I still get the same exception.

**Comment by Eideren** [comment]
Doesn't look like you pushed to the repo, can you try that again ?
Did you add a navigation component, there's also a package for that you should add
`<PackageReference Include="Stride.Navigation" Version="4.1.0.1" />`

**Reply by ly3027929699** [reply]
sorry，I forgot to push. I found that even if I delete all the Scene, it can still run and have SplashUI.

**Reply by ly3027929699** [reply]
I will update the engine to the latest version first. It seems that there is something wrong with the previous version.

**Reply by Eideren** [reply]
You removed `Stride.Core.Assets` and `Stride.Core.Assets.CompilerApp` in this commit https://github.com/ly3027929699/TestStride/commit/bfdf888b87d3bdaa3b28f007e970e3bb6d4c8b80 - if you do so, no new assets will be compiled with your application.
Revert to that commit and add those package references back in, then delete `objs` and `bin` in `GameMenu.Game` and `GameMenu.Windows` folder, and the `TestStride\Bin` folder as well.

**Reply by ly3027929699** [reply]
That’s it, thank you so much.Everything has changed for the better
